docs/gaia/gaia.rst

```diff
@@ -851,6 +852,45 @@ will be able to access to your shared table in a query.
   >>> Gaia.share_table_stop(table_name="user_<user_login_name>.my_table", group_name="my_group")
 
 
+
+---------------------------
+3. Datalink service (Public and Authenticated)
+---------------------------
+
+
+DataLink_ is a data access protocol compliant with the IVOA_ architecture that provides a linking mechanism between
+datasets offered by different services. In practice, it can be seen and used as a web service providing the list of additional
+data products available for each object outside the main catalogue(s). For more information about the products served via
+DataLink in the Gaia ESA Archive we recommend to read the Archive DataLink tutorials available at https://www.cosmos.esa.int/web/gaia-users/archive/datalink-products.
+
+The DataLink products are publicly accessible via Astroquery.Gaia using the *load_data* method. The following example shows how to retrieve the DataLink products associated to three sources in Gaia DR3:
+
+
+.. code-block:: python
+
+  >>> retrieval_type = 'ALL'          # Options are: 'EPOCH_PHOTOMETRY', 'MCMC_GSPPHOT', 'MCMC_MSC', 'XP_SAMPLED', 'XP_CONTINUOUS', 'RVS', 'ALL'
+  >>> data_structure = 'COMBINED'     # Options are: 'INDIVIDUAL', 'COMBINED', 'RAW'
+  >>> data_release   = 'Gaia DR3'     # Options are: 'Gaia DR3' (default), 'Gaia DR2'
+  >>> datalink = Gaia.load_data(ids=[2263166706630078848, 2263178457660566784, 2268372099615724288], data_release = data_release, retrieval_type=retrieval_type, data_structure = data_structure, verbose = False, output_file = None)
+
+The DataLink products are stored inside a Python Dictionary. Each of its elements (keys) contains a one-element list that can be extracted as follows:
+.. code-block:: python
+
+  >>> dl_keys  = [inp for inp in datalink.keys()]
+  >>> dl_keys.sort()
+  >>> print(f'The following Datalink products have been downloaded:')
+  >>> for dl_key in dl_keys:
+        print(f' * {dl_key}')
```

Comment:
I would say to drop all of this, the example above should be sufficient enough for python users.
```suggestion

```

Reply:
We are happy with your changes.